pkgs/development/compilers/llvm/common/default.nix

```diff
@@ -932,18 +932,12 @@ let
           libcxx = null;
           bintools = bintoolsNoLibc';
           extraPackages = [ ];
-          extraBuildCommands =
-            lib.optionalString (lib.versions.major metadata.release_version == "13") ''
-              echo "-nostartfiles" >> $out/nix-support/cc-cflags
-            ''
-            + mkExtraBuildCommands0 cc;
+          extraBuildCommands = mkExtraBuildCommands0 cc;
         }
         // lib.optionalAttrs (lib.versionAtLeast metadata.release_version "14") {
-          nixSupport.cc-cflags =
-            [ "-nostartfiles" ]
-            ++ lib.optional (
-              lib.versionAtLeast metadata.release_version "15" && stdenv.targetPlatform.isWasm
-            ) "-fno-exceptions";
+          nixSupport.cc-cflags = lib.optional (
+            lib.versionAtLeast metadata.release_version "15" && stdenv.targetPlatform.isWasm
+          ) "-fno-exceptions";
```

Comment:
Maybe this should be its own commit with its own explanation?  The motivation is that this was added for pkgsLLVM, but doesn't seem to be necessary any more and is causing problems where clang errors on arguments that don't do anything.

Reply:
Still no explanation in the commit message, and I think it needs it.

Reply:
Commit or comment? I left a comment.

Reply:
Either is fine, as long as it's explained somewhere.

Reply:
Done